html/browsers/the-window-object/window-indexed-properties-strict.html

```diff
@@ -23,6 +23,7 @@
   assert_throws(new TypeError(), function() {
     window[0] = "foo";
   });
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { value: "bar" }))
```

Comment:
Shouldn't we also be testing properties which don't have anything in them, as that was part of the original bug report?